<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>223</width>
      <height>166</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ID producto</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Nombre producto</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Categor√≠a producto</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Proveedor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Stock inicial</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Stock de seguridad</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEditProductId"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditProductName"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxProductCategory"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxProductSupplier"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditProductInitialStock"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditProductBufferStock"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>254</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonProductAdd">
       <property name="text">
        <string>Agegar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonProductModify">
       <property name="text">
        <string>Modificar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonProductDelete">
       <property name="text">
        <string>Eliminar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidgetProduct">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>681</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>ID producto</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nombre producto</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Categoria producto</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Proveedor</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Stock inicial</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Stock de seguridad</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>250</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar tabla</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumain">
    <property name="title">
     <string>Producto</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionagregar"/>
    <addaction name="actionActualizarProducto"/>
    <addaction name="actionEliminarProducto"/>
   </widget>
   <addaction name="menumain"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionagregar">
   <property name="text">
    <string>agregar</string>
   </property>
  </action>
  <action name="actionActualizarProducto">
   <property name="text">
    <string>Actualizar Producto</string>
   </property>
  </action>
  <action name="actionEliminarProducto">
   <property name="text">
    <string>Eliminar Producto</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
